### PR TITLE
INT - update readme further for docker files

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

File: discovery/README.md [hunk 1]
```diff
@@ -2,50 +2,18 @@
...
-| --------------- | -------------------------------------------------------------- | ----------------------------- |
-| dep             | downloads all dependencies needed to build the discovery agent | /vendor                       |
-| test            | runs go test against all test files int he repo                | test results                  |
-| update-sdk      | pulls the latest changes to main on the SDK repo               |                               |
-| build           | builds the binary discovery agent                              | bin/apigee_discovery_agent    |
-| apigee-generate | generates the models for the Apigee APIs                       | pkg/apigee/models             |
-| docker-build    | builds the discovery agent in a docker container               | apigee-discovery-agent:latest |
-
-### Build (Docker)
+The Apigee Discovery Agent is built and distributed as a docker image. To execute the agent with the image the following command may be executed, updating the version.
```
Comment (sbolosan) [inline discovery/README.md:7]: the second sentence seems klunky to me...

"To execute the agent using the image, the following command may be executed.  The result is an updated version."

something like that?

File: discovery/README.md [hunk 2]
```diff
@@ -2,50 +2,18 @@
...
 
 ```shell
-make docker-build
+docker run --env-file env_vars -v `pwd`/keys:/keys ghcr.io/axway/apigee_discovery_agent:v0.1.31
 ```
 
-### Run (Docker)
+If using a local specs path mount into the /specs volume and set the `APIGEE_SPECCONFIG_LOCALPATH` variable to `/specs`.
```
Comment (sbolosan) [inline discovery/README.md:13]: this  one too maybe?
"If using a local specs path, mount to the /specs volume and set...

File: discovery/README.md (hunk 1 above)
Comment (jcollins-axway) [reply discovery/README.md:7]: "The Apigee Discovery Agent is built and distributed as a docker image. To execute the agent using the image, the following command may be executed. Update the version in the examples below."

Comment (sbolosan) [reply discovery/README.md:7]: i like it...

**Review by sbolosan — APPROVED** [review]

**Pull request merged**